tests/test_chromatic.py

```diff
@@ -1717,6 +1717,31 @@ def test_ChromaticAiry():
     t2 = time.time()
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
+def test_chromatic_fiducial_wavelength():
+    """ Check that chromatic code can handle profiles with flux(effective_wavelength) = 0.
+    """
+    import time
+    t1 = time.time()
+
+    waves = np.arange(500., 600.1, 10.)
+    blue_flux = waves < 550.0
+    red_flux = waves > 550.0
+    bp = galsim.Bandpass(galsim.LookupTable(waves, waves**0, interpolant='linear'))
+    blue_sed = galsim.SED(galsim.LookupTable(waves, blue_flux, interpolant='linear'))
+    red_sed = galsim.SED(galsim.LookupTable(waves, red_flux, interpolant='linear'))
+
+    gal1 = galsim.Gaussian(fwhm=1) * blue_sed
+    gal2 = galsim.Gaussian(fwhm=1) * red_sed
+    img1 = gal1.drawImage(bp)
+    img2 = gal2.drawImage(bp)
+    # Didn't see an obvious np.testing method for checking finiteness, so just use assert.
```

Comment:
I find the code easier to read with assert, but the error message when there's a problem is a bit more informative / easier to read with np.testing (just in my opinion).  For example, consider

```
x = 1
y = 3
assert x==y,'badness happened'
```
which gives the following:
```
AssertionError                            Traceback (most recent call last)
/Users/rmandelb/<ipython-input-6-1ea81ded12f9> in <module>()
----> 1 assert x==y,'badness happened'

AssertionError: badness happened
```

This tells me the code that failed.  But if I had done
```
np.testing.assert_equal(x,y,err_msg='badness happened')
```
I would have gotten
AssertionError: 
```
Items are not equal: badness happened
 ACTUAL: 1
 DESIRED: 3
```
which tells me how wrong things were, instead of just that they were wrong.